reducers/plugins/index.ts

```diff
@@ -294,10 +309,10 @@ function adminConsoleCustomComponents(state = {}, action) {
         const key = action.data.key.toLowerCase();
 
         const nextState = {...state};
-        if (!nextState[pluginId]) {
-            nextState[pluginId] = {};
+        let nextArray: any = {};
```

Comment:
Do we need to type this? If so, can we remove any and try this?
```suggestion
        let nextArray: Record<string, AdminConsolePluginComponent> = {};
```